tests/llmcompressor/transformers/kv_cache/test_kv_cache.py

```diff
@@ -232,12 +226,24 @@ def test_kv_cache_gptq_model_state_dict_attr(kv_cache_fixture, tmp_path):
                             strategy: "channel"
                             actorder: False
                         targets: ["Linear"]
+                kv_cache_scheme:
+                    num_bits: {num_bits}
+                    type: {_type}
+                    strategy: {strategy}
+                    dynamic: {dynamic}
+                    symmetric: {symmetric}
     """
 
     output_dir, _ = next(kv_cache_fixture(recipe, tmp_path))
 
     with init_empty_weights():
-        model = AutoModelForCausalLM.from_pretrained(output_dir)
+        # TODO: There is a bug in `apply_quantization_config` which means that, if using
+        # CompressedLinears, the compression status is inferred to `compressed` and
+        # therefore the attention kvcache parameters never undergo initializations
```

Comment:
In other words, we can load quantized kv cache for decompression post compressing, correct? 
Not really a bug - more so this feature was never built out on purpose. We can discus on slack

Reply:
If this was intended behavior, then we should throw an explicit error rather than silently failing to load parameters